src/utils/helpers.js

```diff
@@ -0,0 +1,6 @@
+let key = 0;
+
+export const randomKey = ()  => {
+    key++;
```

Comment:
This function for sure does not generate a random number, please do more research on this

Reply:
noted.